### ISSUE TITLE
#2899849------------- Elements must have sufficient color contrast

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#529]
<strong>Source:</strong>`<span class="c-coordinateJobPanel__gridFieldText ng-star-inserted" title="WMX User">WMX User</span>`
<strong>Element Location: </strong>.p-highlight.p-selectable-row > .ui-resizable-column:nth-child(1) > .c-coordinateJobPanel__gridFieldText[title="WMX User"]
<strong>Impact: </strong>serious
<strong>failureSummary:</strong> Fix any of the following:
  Element has insufficient color contrast of 4.49 (foreground color: #6a6a6a, background color: #d4eefe, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1
Tags: cat.color,wcag2aa,wcag143,ACT
Screenshot:`C:\Users\khy12817\Desktop\Test.PNG`
